README.md

```diff
@@ -1,135 +1,104 @@
 
 # Model the process
 
-In this example, we'll model a simple process to send an email message.
+In this example, we'll model a process to send an email message.
 
-Camunda Platform 8 allows you to use the
+Camunda 8 allows you to use the
 [BPMN](https://docs.camunda.io/docs/components/modeler/bpmn/bpmn-primer/)
 standard to model your business processes. The process consists of two
-tasks: a user task to allow a human to enter the message content, and a service
-task to automatically send the email message.
+tasks: a [user task](https://docs.camunda.io/docs/next/guides/orchestrate-human-tasks/) to allow a human to enter the message content, and a [service
+task](https://docs.camunda.io/docs/next/guides/orchestrate-microservices/) to automatically send the email message.
 
 ![Process](images/send-email.png)
 
-**Note**: To keep the scope of this guide easy to follow, we only build the
-scaffolding for this process automation example and will not actually send
-an email. This can be a task for you to extend the example and connect to a mail
-provider.
-
-To create the process, we use the desktop [Camunda Modeler] or [Web Modeler].
+**NOTE: To keep the scope of this guide limited, this guide only steps through the framework for this process automation example and will not actually send an email. If you would like to send an email, you may extend the example and [connect to a mail provider](https://docs.camunda.io/docs/next/guides/configuring-out-of-the-box-connectors/).**
 
-Find the finished process at [process/send-email.bpmn](process/send-email.bpmn).
+To create the process, use [Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/).
 
-## Model the user task form
+Find the finished process at [process/send-email.bpmn](process/send-email.bpmn), and learn more about [designing a process using BPMN](https://docs.camunda.io/docs/next/guides/automating-a-process-using-bpmn/).
 
-To help users complete the user task **Enter Message**, we create a user
-task form using the modeler.
+## Model and configure the user task form
 
-Use [Camunda Modeler] or [Web Modeler] to create the form.
+To help users complete the user task **Enter message**, [create a user
+task form using Modeler](https://docs.camunda.io/docs/next/guides/utilizing-forms/).
 
-Find the finished form at [process/enter-email-message.form](process/enter-email-message.form).
+Find the finished form at [process/enter-email-message.form](process/enter-email-message.form)
 
-## Configure user task to use form
-
-Now that we've created a form, we must configure the user task to use the form.
-To do this, we copy the JSON content of the form into the **Form Json
-Configuration** field under the **Forms** tab of the modeler.
+Configure the user task to use the form.
+To do this, copy the JSON content of the form into the **Form JSON configuration** field under the **Form** tab in the Modeler properties panel.
```

Comment:
I didn't have to do this if I uploaded the BPMN file. Could be worth noting.

Reply:
I actually would prefer to make use of form linking instead as this c&p of form json is not really a great experience in general. But if the form is already part of the sample file we may omit this part completely?

Reply:
wdyt of just instructing the use to upload the form and the process model (which links to the form)? Basically not getting into modeling functionalities at all here? 🤔